CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## teradata_utils 1.3.0
+
```

Comment:
We need to add insert_by_period materialization as a feature in this changelog.md file Write Preview Cancel Update comment